openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -22,12 +22,33 @@
 
 def get_profile_image_storage():
     """
-    Configures and returns a django Storage instance that can be used
-    to physically locate, read and write profile images.
-    """
-    config = settings.PROFILE_IMAGE_BACKEND
-    storage_class = get_storage_class(config['class'])
-    return storage_class(**config['options'])
+    Returns an instance of the configured Django storage class for profile images.
+
+    The function looks for the `PROFILE_IMAGE_BACKEND` setting in the Django settings.
+    If it exists and includes a `'class'` key, that class is used as the storage backend.
+    If the setting is missing or does not include a `'class'`, the default storage backend
+    DEFAULT_FILE_STORAGE will return.
+
+    Returns:
+        An instance of the configured storage backend.
+    """
+    config = getattr(settings, 'PROFILE_IMAGE_BACKEND', {})
+    storage_class_path = config.get('class')
+    options = config.get('options', {})
+
+    if not storage_class_path:
+        storage_class_path = (
+            getattr(settings, 'STORAGES', {}).get('profile_images', {}).get('BACKEND') or # named storages
```

Comment:
`profile_images ` it does not exists right now so it will use following storages.